### PR TITLE
changes.md: update for 1.7.1-tectonic.1

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: changes.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 
-## Current
+## Tectonic 1.7.1-tectonic.1 (2017-08-07)
+
+* Updates to Kubernetes v1.7.1.
+* Support for Azure is Beta.
 
 ### Console
 
+* The Tectonic update channel can now be selected (1.6, 1.7-preproduction, 1.7-production)
```
Comment (aaronlevy) [inline changes.md]: Maybe put a warning that you should first upgrade to `v1.6.7_tectonic.2` before trying to upgrade to v1.7?

Comment (Quentin-M) [inline changes.md]: Absolutely, I mentioned it earlier in Slack and forgot to include it there, thanks! Maybe @robszumski you'd would like to input your own wording?

File: changes.md [hunk 2]
```diff
@@ -1,15 +1,24 @@
 # Tectonic Changelog
 
-## Current
+## Tectonic 1.7.1-tectonic.1 (2017-08-07)
+
+* Updates to Kubernetes v1.7.1.
+* Support for Azure is Beta.
```
Comment (robszumski) [inline changes.md]: note that this isn't confirmed yet

**Comment by Quentin-M** [comment]
Thank you for your help on this!

**Comment by Quentin-M** [comment]
Could you please tell me which link? We never included any link in `changes.md`.

**Comment by Quentin-M** [comment]
@robszumski Updated :)

**Comment by Quentin-M** [comment]
@robszumski Are we going with it? 👍

File: changes.md [hunk 3]
```diff
@@ -1,15 +1,34 @@
 # Tectonic Changelog
 
-## Current
+## Tectonic 1.7.1-tectonic.1 (2017-08-07)
```
Comment (athai) [inline changes.md]: Should the date be updated?

**Comment by Quentin-M** [comment]
@robszumski docs using this file are sync'd. we can merge it~

**Pull request merged**